_health-care/_js/_components/mothers_maiden_name.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import _ from 'lodash';
+
+class MothersMaidenName extends React.Component {
+  constructor() {
+    super();
+    this.state = {hasError: false};
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  componentWillMount() {
+    this.id = _.uniqueId();
+  }
+
+  handleChange() {
+    const name = this.refs.name.value;
+
+    if (!this.validate(name)) {
+      this.setState({hasError: true});
+    } else {
+      this.setState({hasError: false});
+    }
+
+    this.props.onUserInput(name);    
+  }
+
+  validate(field) {
+    if (field === '') {
```

Comment:
In another PR, we should create a validations.js module with a "validNameField" function so that this and MothersMaidenName can just share the same regex logic. It'll also centralize the unittest.